openmp/runtime/src/include/ompx.h.var

```diff
@@ -9,13 +9,21 @@
 #ifndef __OMPX_H
 #define __OMPX_H
 
-#ifdef __AMDGCN_WAVEFRONT_SIZE
-#define __WARP_SIZE __AMDGCN_WAVEFRONT_SIZE
-#else
-#define __WARP_SIZE 32
+#if (defined(__NVPTX__) || defined(__AMDGPU__))
+#include <gpuintrin.h>
+#define __OMPX_TARGET_IS_GPU
 #endif
 
 typedef unsigned long uint64_t;
+typedef unsigned int uint32_t;
+
+static inline uint32_t __warpSize(void) {
+  #ifdef __OMPX_TARGET_IS_GPU
+    return __gpu_num_lanes();
+  #else
+    __builtin_trap();
+  #endif
```

Comment:
```suggestion
#ifdef __OMPX_TARGET_IS_GPU
  return __gpu_num_lanes();
#else
  __builtin_trap();
#endif
```

Reply:
I believe if you use clang-format on this file, the macro would be pushed all the way to the beginning of the file.